tests/tests_insights_client_register.yml

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Basic insights-client register/unregister
+  hosts: all
+  gather_facts: true
+  become: true
+  tags:
+    - tests::slow
+  tasks:
+    - name: Register insights
+      include-role:
```

Comment:
Basically, in ansible, everything on the left hand side of the `:` must be `[a-zA-Z0-9_]` - no `-` allowed. (This isn't strictly true, but it is a good rule of thumb to remember to use `foo_bar:` instead of `foo-bar:`